src/components/modals/CreateRepoModalTemplate.tsx

```diff
@@ -213,4 +228,5 @@ type CreateRepositoryModalTemplateProps = {
   isModalOpen: boolean;
   handleModalToggle?: () => void;
   orgName?: string;
+  updateListHandler: (value: IRepository) => void;
```

Comment:
don't think we use this anywhere

Reply:
what about this one? https://github.com/quay/quay-ui/blob/main/src/routes/RepositoriesList/RepositoriesList.tsx#L260

Reply:
yeah, it's a dead code. I forgot to remove it in my earlier PR.